### PR TITLE
Inline ADVISORY_LOCK_PREFIX_* values

### PR DESCRIPTION
**Pull request by JamesGuthrie**
## Description

These placeholders were replaced during the migration process. When we
move these SQL scripts to the extension we would either need to
replicate this, or just inline the values here. Inlining seems to be the
simpler solution.

## Merge requirements

Please take into account the following non-code changes that you may need to make with your PR:

- [-] CHANGELOG entry for user-facing changes
- [-] Updated the relevant documentation

### CODE REVIEW

**Review by cevian — APPROVED** [review]

File: pkg/migrations/sql/idempotent/base.sql [hunk 1]
```diff
@@ -2754,7 +2762,7 @@ BEGIN
         WHERE h.schema_name = 'SCHEMA_DATA' and h.table_name = ht_table;
 
         --alter job schedule is not currently concurrency-safe (timescaledb issue #2165)
-        PERFORM pg_advisory_xact_lock(ADVISORY_LOCK_PREFIX_JOB, bgw_job_id);
+        PERFORM pg_advisory_xact_lock(12377, bgw_job_id);
```
Comment (jgpruitt) [inline pkg/migrations/sql/idempotent/base.sql:2765]: Why did we not create a function for this one? Is it because this is the only instance of the number?

**Review by jgpruitt — CHANGES_REQUESTED** [review]

File: pkg/migrations/sql/idempotent/base.sql (hunk 1 above)
Comment (JamesGuthrie) [reply pkg/migrations/sql/idempotent/base.sql:2765]: That was the idea. I can also pull this into a function if you would prefer.

Comment (JamesGuthrie) [reply pkg/migrations/sql/idempotent/base.sql]: I've pulled this into a function

**Review by jgpruitt — APPROVED** [review]
lgtm!

**Pull request merged**